src/FSharpFakeTargets/targets.fs

```diff
@@ -226,6 +235,14 @@ module Targets =
            p.AssemblyInfoFilePaths |> Seq.iter (_map setMeta)
          )
 
+  let _determineTestTarget parameters =
```

Comment:
Let's call this _testTarget, and rename the previous _testTarget as _nunitTestTarget maybe?